README.md

```diff
@@ -56,6 +56,11 @@ CI (Continuous Integration) integrates code changes into a common branch of a sh
 
 CD (Continuous Delivery) is the practice of delivering code changes to production frequently and reliably. An example would be using GitHub Actions to run tests via CI, prepare a docker image for the app once the tests pass, and then trigger the deployment via CD.
 
+### CDN
```

Comment:
@beingbravado Can you remove this? This is not really talking about o11y?

Reply:
Done